modules/contact/test/tests/tan-pen-and-scaling/tests

```diff
@@ -0,0 +1,22 @@
+[Tests]
+  issues = '#12601'
+  design = 'Constraints/index.md'
+  [auto-scaling]
+    type = Exodiff
+    input = bouncing-block-tan-pen.i
+    exodiff = bouncing-block-tan-pen_exo.e
+    requirement = 'The system shall be able to apply automatic scaling in conjunction with tangential penalty contact constraint enforcement and show zero penetration with a unity (normal) penalty factor'
+    abs_zero = 1e-8
+    cli_args = "Postprocessors/active=''"
+    rel_err = 7e-5
+  []
+  [no-scaling]
+    type = Exodiff
+    input = bouncing-block-tan-pen.i
+    exodiff = no-scaling.e
+    requirement = 'The system shall show penetration when automatic scaling is not used with tangential penalty contact constraint enforcement and when the element Jacobian diagonals are significantly larger than the (normal) penalty factor'
+    cli_args = "Executioner/automatic_scaling=false Outputs/file_base=no-scaling Postprocessors/active='' -pc_type lu -pc_factor_shift_type NONZERO"
```

Comment:
Why do you need `-pc_factor_shift_type NONZERO` on problems like this?

Reply:
> Why do you need `-pc_factor_shift_type NONZERO` on problems like this?

When things weren't working well I needed to use `lu` in order for this to converge in parallel. But now that things are hunky dorey, I've removed it :smile:

Reply:
Good to hear!